### ISSUE TITLE
[BUG]: A11Y Checker not handling the case of multiple headers for both rows and columns.

### ISSUE THREAD

**Issue by fbarroso24** [IBMa/equal-access#1472]
### Project

a11y checker

### Browser

Chrome

### Operating system

Windows

### Description

We found a bug in `Accessibility checker` tool.

We have a complex table where multiple rows & columns act as headers (see screenshot). 
![image](https://github.com/IBMa/equal-access/assets/8657768/4c134928-10a6-46cd-a485-82c8a6c2550f)

 The `Accessibility Checker` is showing a `1.3.1: Info & Relationships` violation wrt `Table has no headers identified`.

That being said, we already appear to be implementing the proposed solution by using `id` attribute in the `<th>` elements and the `headers` attribute in the `td` cells. (see above pic).

Given the above, we believe this is a false-positive violation due to the `A11Y checker` not handling the case of multiple headers for both rows and columns. 

Here is a sample HTML to help illustrate the scenario.
[complexTable.zip](https://github.com/IBMa/equal-access/files/11566002/complexTable.zip)

It's essentially the raw HTML to represent a complex chart like. 
![image](https://github.com/IBMa/equal-access/assets/8657768/ec5ab325-0324-49ca-a74b-766f97de40c7)
NOTE: Given that .js & .css files are not included, the output of the sample HTML file will look a bit different.

### Steps to reproduce

Run `A11Y checker` on a complex table that includes multiple header rows & multiple header columns.
[complexTable.zip](https://github.com/IBMa/equal-access/files/11566043/complexTable.zip)

**Comment by philljenkins** [comment]
See the discussion on Slack at https://ibm-studios.slack.com/archives/C036P1CTN/p1684856928127849
also the separate reply:
_"We're going to have to dig a little more. Data tables are generally not expected to have the headers on the bottom."_

**Comment by shunguoy** [comment]
triage: the current rule only allows the headers in the first row, and empty cell in the left.  also need to consider when table header is in the bottom of the data table.

**Comment by shunguoy** [comment]
for @tombrunet review

**Comment by ErickRenteria** [comment]
@shunguoy unable to test due to https://github.com/IBMa/equal-access/issues/1684

**Comment by ErickRenteria** [comment]
This seem to be working correctly now. 
![Screenshot 2023-09-18 115248](https://github.com/IBMa/equal-access/assets/62436670/a69a2cea-a6c8-4405-b2e1-fc41d66546b6)
